code/modules/mob/living/carbon/xenomorph/life.dm

```diff
@@ -78,37 +78,37 @@
 	if(recovery_aura)
 		sunder_recov *= 1 + recovery_aura * 0.1 //10% bonus per rank of recovery aura
 
-	SEND_SIGNAL(src, COMSIG_XENOMORPH_SUNDER_REGEN, seconds_per_tick)
+	SEND_SIGNAL(src, COMSIG_XENOMORPH_SUNDER_REGEN, src)
 
 	adjust_sunder(sunder_recov)
 
-/mob/living/carbon/xenomorph/proc/handle_critical_health_updates(seconds_per_tick)
+/mob/living/carbon/xenomorph/proc/handle_critical_health_updates()
 	if(loc_weeds_type)
-		heal_wounds(XENO_RESTING_HEAL, seconds_per_tick)
```

Comment:
This is a revert PR, but commenting here instead of the original PR:

The unreverted line should be `heal_wounds(XENO_RESTING_HEAL, FALSE, seconds_per_tick)` since as a 2-argument call to a 3-parameter function, it would be interpreted as `heal_wounds(XENO_RESTING_HEAL, seconds_per_tick, null)`. This seems the likely cause of xenos not healing in crit.

Reply:
damn, thanks, will fix